test/e2e/e2e_test.go

```diff
@@ -99,85 +99,121 @@ func RcByNameContainer(name string, replicas int32, labels map[string]string, gr
 
 // startEndToEndForLowNodeUtilization tests the lownode utilization strategy.
 func startEndToEndForLowNodeUtilization(ctx context.Context, clientset clientset.Interface, nodeInformer coreinformers.NodeInformer) {
-	var thresholds = make(deschedulerapi.ResourceThresholds)
-	var targetThresholds = make(deschedulerapi.ResourceThresholds)
-	thresholds[v1.ResourceMemory] = 20
-	thresholds[v1.ResourcePods] = 20
-	thresholds[v1.ResourceCPU] = 85
-	targetThresholds[v1.ResourceMemory] = 20
-	targetThresholds[v1.ResourcePods] = 20
-	targetThresholds[v1.ResourceCPU] = 90
 	// Run descheduler.
 	evictionPolicyGroupVersion, err := eutils.SupportEviction(clientset)
 	if err != nil || len(evictionPolicyGroupVersion) == 0 {
 		klog.Fatalf("%v", err)
 	}
-	stopChannel := make(chan struct{})
-	nodes, err := nodeutil.ReadyNodes(ctx, clientset, nodeInformer, "", stopChannel)
+
+	nodes, err := nodeutil.ReadyNodes(ctx, clientset, nodeInformer, "", nil)
 	if err != nil {
 		klog.Fatalf("%v", err)
 	}
 
-	lowNodeUtilizationStrategy := deschedulerapi.DeschedulerStrategy{
-		Enabled: true,
-		Params: &deschedulerapi.StrategyParameters{
-			NodeResourceUtilizationThresholds: &deschedulerapi.NodeResourceUtilizationThresholds{
-				Thresholds:       thresholds,
-				TargetThresholds: targetThresholds,
+	strategies.LowNodeUtilization(
+		ctx,
+		clientset,
+		deschedulerapi.DeschedulerStrategy{
+			Enabled: true,
+			Params: &deschedulerapi.StrategyParameters{
+				NodeResourceUtilizationThresholds: &deschedulerapi.NodeResourceUtilizationThresholds{
+					Thresholds: deschedulerapi.ResourceThresholds{
+						v1.ResourceMemory: 20,
+						v1.ResourcePods:   20,
+						v1.ResourceCPU:    85,
+					},
+					TargetThresholds: deschedulerapi.ResourceThresholds{
+						v1.ResourceMemory: 20,
+						v1.ResourcePods:   20,
+						v1.ResourceCPU:    90,
+					},
+				},
 			},
 		},
-	}
-
-	podEvictor := evictions.NewPodEvictor(
-		clientset,
-		evictionPolicyGroupVersion,
-		false,
-		0,
 		nodes,
-		false,
+		evictions.NewPodEvictor(
```

Comment:
could you pass the podEvictor created in `evictPods` to this function and use that?